src/management/src/management_frontend.hpp

```diff
@@ -670,6 +670,7 @@ class RequestHandler {
     InputType input_type = clipper::parse_input_type(input_type_raw);
     std::string container_name = get_string(d, "container_name");
     std::string model_data_path = get_string(d, "model_data_path");
+    boost::optional<int> batch_size = get_int(d, "batch_size");
```

Comment:
Add a check to ensure that `batch_size` is either `-1` or a positive integer, and throw a `ManagementOperationError` if this condition is not met.

Reply:
OK. I will change it as soon as possible.